app/views/content/life-as-a-teacher/change-careers/_categories.html.erb

```diff
@@ -0,0 +1,28 @@
+<div class="row">
+  <section class="category col-space-m">
+    <section class="container category__cards main-section">
+      <nav class="category__nav-cards">
+        <ul>
+          <%= render Categories::CardComponent.new(heading_tag: "h2", card:
+            OpenStruct.new(
+              title: "Support with changing careers to teaching",
+              description: "Discover the support and resources available to help you transition smoothly into a rewarding teaching career.",
+              path: "/life-as-a-teacher/change-careers/support-with-changing-careers"
+            )) %>
+          <%= render Categories::CardComponent.new(heading_tag: "h2", card:
+            OpenStruct.new(
+              title: "Benefits of changing careers to teaching",
+              description: "Learn about the benefits of changing careers to teaching. From competitive salaries and job security, to making a  real difference in young people's lives.",
+              path: "/life-as-a-teacher/change-careers/benefits-of-changing-careers"
+            )) %>
+          <%= render Categories::CardComponent.new(heading_tag: "h2", card:
+            OpenStruct.new(
+              title: "How to change careers to become a teacher",
+              description: "Find out the key steps to take to change careers to become a teacher.",
```

Comment:
This is slightly harder to read with the repeated `to`s - is it possible to reword slightly?